examples/animation/multiple_axes.py

```diff
@@ -0,0 +1,79 @@
+"""
+=======================
+Multiple axes animation
+=======================
+
+This example showcases:
+
+- how animation on subplots works,
```

Comment:
```suggestion
- how animation on multiple subplots works,
```

Reply:
Fixed.  
I took the description straight from the linked issue, but inserting `multiple` as in the filename makes it clearer.